### ISSUE TITLE
issue with geckodriver when running inside a dockerized python app

### ISSUE THREAD

**Issue by Emirov** [shaikhsajid1111/facebook_page_scraper#24]
I work with a fast api application this worked with me on the local but when tried to "dockerize" the app got this exception 
```
  raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: Expected browser binary location, but unable to find binary in default location, no 'moz:firefoxOptions.binary' capability provided, and no binary flag set on the command li
```
It seems to be an exception related to the path maybe of geckodriver

**Comment by shaikhsajid1111** [comment]
Yeah, it tries to find Mozilla Firefox in default location of installation but it is unable to find so

**Comment by Emirov** [comment]
Thanks please view too my solution for chrome too but don't consider the push that I tried because there is an issue in it

**Comment by shaikhsajid1111** [comment]
Oh, great if that works for you. Yeah, I checked it looks good, I'll test than might consider that. Thank you

**Comment by Emirov** [comment]
Thanks Sajid waiting for the fix!
    Le jeudi 1 septembre 2022, 10:37:10 UTC+1, Sajid Shaikh ***@***.***> a écrit :  
 
 


Oh, great if that works for you. Yeah, I checked it looks good, I'll test than might consider that. Thank you
